### PR TITLE
474 billboard client

### PR DESCRIPTION
**Pull request by AlexNi245**

### CODE REVIEW

File: packages/billboard-client/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+config.yml
```
Comment (hai-ko) [inline packages/billboard-client/.gitignore:2]: Do we have a config.yml?

File: packages/billboard-client/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+yarn
+```
+yarn start
+```
+
+npm 
+```
+yarn start
```
Comment (hai-ko) [inline packages/billboard-client/README.md:21]: Should be `npm start`

File: packages/billboard-client/package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+    "name": "dm3-billboard-client",
+    "version": "0.1.0",
+    "private": true,
```
Comment (hai-ko) [inline packages/billboard-client/package.json:4]: Not sure if the repo should be private

File: packages/billboard-client/test/helper/mockDeliveryServiceHttpClient.ts
Comment (hai-ko) [inline packages/billboard-client/test/helper/mockDeliveryServiceHttpClient.ts:1]: Can't this be deleted?

**Review by hai-ko — CHANGES_REQUESTED** [review]

File: packages/billboard-client/.gitignore (hunk 1 above)
Comment (AlexNi245) [reply packages/billboard-client/.gitignore:2]: No i just copied it from be will remove this part

**Review by hai-ko — APPROVED** [review]

**Pull request merged**